client/src/components/styledText.css

```diff
@@ -95,6 +96,16 @@
     filter: drop-shadow(0 0 .2em var(--spiral-shine))
             drop-shadow(.1rem .1rem .1rem rgb(0, 0, 0));
 }
+.keyword-poly {
+    /*
+        Not scrolling because I looked it up and one of the reasons pi is on there is that is pi is greek letter for p so p should be yellow
```

Comment:
What does this have to do with whether it scrolls?

Remove comment - even if it is relevant, this is not an implementation detail, it's a design detail